packages/flame/lib/src/test_helpers/random_test.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:math';
+import 'package:meta/meta.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+final _rnd = Random(); // for generating seeds
+const _maxSeed = 4294967296;
+
+/// This function is equivalent to `test(name, body)`, except that it is
+/// better suited for randomized testing: it will create a Random
+/// generator and pass it to the test body, but also record the seed
+/// that was used for creating the random generator. Thus, if a test
+/// fails for a specific rare seed, it would be easy to reproduce this
+/// failure.
+///
+/// In order for this to work properly, all random number generation
+/// within `testRandom()` must be performed through the provided random
+/// generator.
+///
+/// Example of use:
+///
+///     testRandom('description', (Random random) {
+///       expect(random.nextDouble() == random.nextDouble(), false);
+///     });
+///
+/// Then if the test output shows that the test failed with seed `S`,
+/// simply adding parameter `seed=S` into the function will force it
+/// to run for that specific seed.
+///
```

Comment:
Noticed u always put an empty doc line in your supplied docs.

Havent had a "nit" about it yet but it isn't according to our (unwritten) dartdoc standards :-). Any reason why u do this?

Reply:
No problem, I'll remove the extra line.

As for the reason, I normally code in C++ / Python, and in C++ the doc-comments look like this:
```
/**
 * doc...
 */
```
while  in Python they are
```
"""
doc...
"""
```
so in both cases, there is a visual empty line after the doc-comment